service/controller/v10/cluster_resource_set.go

```diff
@@ -99,6 +100,9 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 	if config.ProjectName == "" {
 		return nil, microerror.Maskf(invalidConfigError, "config.ProjectName must not be empty")
 	}
+	if config.APIWhitelist.Enabled && config.APIWhitelist.SubnetList == "" {
+		return nil, microerror.Maskf(invalidConfigError, "%T.APIWhitelist.SubnetList must not be empty when %T.APIWhitelist is enabled", config)
+	}
```

Comment:
SubnetList could be further validated, but it isn't only thing not being checked.

Reply:
We should work towards the `Config.Validate() error` thingy, but not in this PR. Just saying to raise awareness again.